modules/core/arrow-test/src/main/kotlin/arrow/test/laws/TraverseLaws.kt

```diff
@@ -121,4 +125,14 @@ object TraverseLaws {
       val mapped = fa.foldMap(Int.monoid(), f)
       mapped.equalUnderTheLaw(traversed, Eq.any())
     }
+
+  fun <F, A> Traverse<F>.effectOrderPreserved(GEN: Gen<Kind<F, A>>) = IO.concurrent().run {
+    forAll(GEN) { fa: Kind<F, A> ->
+      val foldableOrder = fa.foldLeft(emptyList()) { xs: List<A>, x: A -> xs + x }
```

Comment:
By the way, does some form of `Foldable.toList` exist anywhere?

Reply:
Not that I know of. That'd be an alias for `combineAll(ListK.monoid())` right?